crates/swc/tests/exec.rs

```diff
@@ -144,10 +144,20 @@ fn create_matrix(entry: &Path) -> Vec<Options> {
     })
     .matrix_bool()
     .matrix_bool()
-    .matrix_bool()
+    .matrix(|| {
+        vec![
+            SourceMapsConfig::Bool(false),
+            SourceMapsConfig::Bool(true),
+            SourceMapsConfig::Inline,
+            SourceMapsConfig::Linked(entry.with_extension(format!(
+                "{}.map",
+                entry.extension().unwrap().to_string_lossy(),
+            ))),
+        ]
+    })
     .into_iter()
     .map(
-        |((((target, syntax), minify), external_helpers), source_map)| {
+        |((((target, syntax), minify), external_helpers), source_maps)| {
```

Comment:
Please revert this. The execution test suite is very slow and adding a matrix will prolong it